src/activities/AnshikaMittal/AnshikaMittal.qml

```diff
@@ -0,0 +1,503 @@
+
+ActivityBase {
+    id: activity
+    property color buttonColor: "lightblue"
+    property color onHoverColor: "gold"
+    property color borderColor: "white"
+
+
+
+    onStart: focus = true
+    onStop: {}
+
+
+    pageComponent: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: "#ABCDEF"
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        GCText {
+            //  anchors.centerIn: parent
+            text: "Anshika Mittal's activity"
+            x:870; y:670;
+            fontSize: mediumSize
+        }
+
+        DialogHelp {
+            id: dialogHelp
+            onClose: home()
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level }
+            onHelpClicked: {
+                displayDialog(dialogHelp)
+            }
+            onPreviousLevelClicked: Activity.previousLevel()
+            onNextLevelClicked: Activity.nextLevel()
+            onHomeClicked: activity.home()
+        }
+
+        Bonus {
+            id: bonus
+            Component.onCompleted: win.connect(Activity.nextLevel)
+        }
+
+        Image{
+            id: img1
+
+               width : 200;
+               height: 250;
+                x: 950
+                y:350
+                   source: "qrc:/gcompris/src/activities/AnshikaMittal/resource/blocks.svg"
+                   anchors.fill: parent.BottomLeft
+        }
+
+
+        Rectangle {  id: text_title ; color: "orange"; radius: 40.0
+            width: 400; height: 60
+            x: 500
+            y:20
+            Text { anchors.centerIn: parent
+                font.pointSize: 25; text: "-- Let's make a tower --" }
+
+        }
+
+        TextEdit {
+            id: text_map
+            textFormat: TextEdit.RichText
+            anchors {
+                right: parent.right
+
+                margins: 10
+            }
+            x: 500
+            y: 500
+            font {
+                pointSize: (18 + ApplicationSettings.baseFontSize) * ApplicationInfo.fontRatio
+                capitalization: ApplicationSettings.fontCapitalization
+                weight: Font.DemiBold
+                family: GCSingletonFontLoader.fontLoader.name
+                letterSpacing: ApplicationSettings.fontLetterSpacing
+                wordSpacing: 10
+            }
+
+            function insertText(text) {
+                text_map.insert(parent.BottomLeft, text)
```

Comment:
the first argument is an integer (http://doc.qt.io/qt-5/qml-qtquick-textedit.html#insert-method) not an anchor.